commit 6f7a17fa25436e30d13b0d5da0ff7c721fcbc0dc (HEAD -> 22.04-build)
Author: Justin Cattle <j@ocado.com>
Date:   Thu Aug 10 08:48:33 2023 +0100

    Fix for 22.04 build

diff --git a/Makefile b/Makefile
index 36c43e57..885ccbe1 100644
--- a/Makefile
+++ b/Makefile
@@ -18,7 +18,7 @@
 ##### Global variables #####
 
 WITH_NVCGO   ?= yes
-WITH_LIBELF  ?= no
+WITH_LIBELF  ?= yes
 WITH_TIRPC   ?= no
 WITH_SECCOMP ?= yes
 
@@ -169,6 +169,9 @@ ifeq ($(WITH_TIRPC), yes)
 LIB_CPPFLAGS       += -isystem $(DEPS_DIR)$(includedir)/tirpc -DWITH_TIRPC
 LIB_LDLIBS_STATIC  += -l:libtirpc.a
 LIB_LDLIBS_SHARED  += -lpthread
+else
+LIB_CPPFLAGS       += -DWITH_TIRPC $(shell pkg-config --cflags libtirpc)
+LIB_LDLIBS_SHARED  += $(shell pkg-config --libs libtirpc)
 endif
 ifeq ($(WITH_SECCOMP), yes)
 LIB_CPPFLAGS       += -DWITH_SECCOMP $(shell pkg-config --cflags libseccomp)
